packages/calcite-components/src/components/action-menu/action-menu.tsx

```diff
@@ -45,7 +45,7 @@ export class ActionMenu extends LitElement {
 
   // #region Private Properties
 
-  private guid = `calcite-action-menu-${guid()}`;
+  private guid = IDS.actionMenuId(guid());
```

Comment:
I think we can simplify the name for lookup object utils/keys. For example:
 
```ts
// action-menu
IDS.menuButtonId(foo);
 ```

can be simplified to

```ts
IDS.button(foo);
```

**Reasoning:**

* lookup object name provides context
* resources are scoped to each component, so the component name is implied (also applies to `calcite` namespace used in some resources)

For any IDs applied on the root of the component, we could go with `host`.